packages/loader/container-utils/src/error.ts

```diff
@@ -85,9 +111,37 @@ export class DataProcessingError extends LoggingError implements IErrorBase {
     constructor(errorMessage: string, props?: ITelemetryProperties) {
         super(errorMessage, props);
     }
+
+    /**
+     * Conditionally coerce the throwable input into a DataProcessingError.
+     * @param originalError - Throwable input to be converted.
+     * @param message - Sequenced message (op) to include info about via telemetry props
+     * @returns Either a new DataProcessingError, or (if wrapping is deemed unnecessary) the given error
+     */
+    static wrapIfUnrecognized(
+        originalError: any,
+        message: ISequencedDocumentMessage | undefined,
+    ): ICriticalContainerError {
+        const messagePropsToLog = message !== undefined
+            ? extractSafePropertiesFromMessage(message)
+            : undefined;
+        const newErrorFn =
+            (errMsg: string) =>
+                new DataProcessingError(errMsg);
+
+        // Don't coerce if it's already a recognized LoggingError
+        const error = isValidLoggingError(originalError)
```

Comment:
this code is repeated here and in CreateContainerError why not just have wrapError return the input error if isValidLoggingError, or am i missing cases where we wrap an loggingError

Reply:
i don't see where it is used, but i see how it works in wrap error now

Reply:
Yes there are other cases where we wrap a loggingError.  See ThrottlingWarning and SummarizerError or whatever they're called.